packages/babel-plugin-proposal-decorators/test/fixtures/metadata/subclass-super-decorators copy/exec.js

```diff
@@ -0,0 +1,18 @@
+function dec(v) {
```

Comment:
This file should be named as `subclass-super-decorators copy/exec.js`. 
It is sometimes so annoying that if one duplicates a folder in VSCode, the active folder still points to the original one, so the " copy" folder is still out there while we would have renamed " copy" to "no-super-decorators".